db/schema.rb

```diff
@@ -10,49 +10,48 @@
-    t.integer "user_id", null: false
-    t.string "body", null: false
-    t.datetime "created_at", null: false
-    t.datetime "updated_at", null: false
-    t.integer "commentable_id"
-    t.string "commentable_type"
+  create_table 'comments', force: :cascade do |t|
+    t.integer 'user_id', null: false
+    t.string 'body', null: false
+    t.datetime 'created_at', null: false
+    t.datetime 'updated_at', null: false
+    t.integer 'commentable_id'
+    t.string 'commentable_type'
   end
 
-  create_table "posts", force: :cascade do |t|
-    t.string "title", null: false
-    t.string "link"
-    t.datetime "created_at", null: false
-    t.datetime "updated_at", null: false
-    t.integer "user_id", null: false
+  create_table 'posts', force: :cascade do |t|
+    t.string 'title', null: false
+    t.string 'link'
+    t.datetime 'created_at', null: false
+    t.datetime 'updated_at', null: false
+    t.integer 'user_id', null: false
   end
 
-  create_table "users", force: :cascade do |t|
-    t.string "email", default: "", null: false
-    t.string "encrypted_password", default: "", null: false
-    t.string "reset_password_token"
-    t.datetime "reset_password_sent_at"
-    t.datetime "remember_created_at"
-    t.datetime "created_at", null: false
-    t.datetime "updated_at", null: false
-    t.index ["email"], name: "index_users_on_email", unique: true
-    t.index ["reset_password_token"], name: "index_users_on_reset_password_token", unique: true
+  create_table 'users', force: :cascade do |t|
+    t.string 'email', default: '', null: false
+    t.string 'encrypted_password', default: '', null: false
+    t.string 'reset_password_token'
+    t.datetime 'reset_password_sent_at'
+    t.datetime 'remember_created_at'
+    t.datetime 'created_at', null: false
+    t.datetime 'updated_at', null: false
+    t.index ['email'], name: 'index_users_on_email', unique: true
+    t.index ['reset_password_token'], name: 'index_users_on_reset_password_token', unique: true
   end
 
-  create_table "votes", force: :cascade do |t|
-    t.integer "user_id", null: false
-    t.integer "votable_id", null: false
-    t.string "votable_type", null: false
-    t.integer "vote_type", null: false
-    t.datetime "created_at", null: false
-    t.datetime "updated_at", null: false
-    t.index ["user_id", "votable_type", "votable_id"], name: "index_votes_on_user_id_and_votable_type_and_votable_id", unique: true
+  create_table 'votes', force: :cascade do |t|
+    t.integer 'user_id', null: false
+    t.integer 'votable_id', null: false
+    t.string 'votable_type', null: false
+    t.integer 'vote_type', null: false
+    t.datetime 'created_at', null: false
+    t.datetime 'updated_at', null: false
+    t.index %w[user_id votable_type votable_id], name: 'index_votes_on_user_id_and_votable_type_and_votable_id', unique: true
   end
 
-  add_foreign_key "posts", "users", on_delete: :cascade
+  add_foreign_key 'posts', 'users', on_delete: :cascade
 end
```

Comment:
You should configure rubocop to ignore this whole file. It's auto-generated and shouldn't be messed with.